int32 brightness
---
int32 error